editoast/openapi.yaml

```diff
@@ -2546,6 +2546,21 @@ components:
             is_warning:
               type: boolean
 
+    TrackSectionLocation:
+      type: object
+      description: A track location (track section and offset)
+      properties:
+        track_section:
+          type: string
+          description: The track section ID
+          example: 61205924-6667-11e3-81ff-01f464e0362d
+        offset:
+          type: number
+          format: float
+          description: The offset on the track section
+          example: 42.
+      required: [track_section, offset]
+
     TrackLocation:
       type: object
       description: A track location (track section and offset)
```

Comment:
These are the same objects ?

Reply:
They're not exactly the same. We should be more consistent (but later)